main/depend.common

```diff
@@ -685,6 +685,7 @@ module_pbl_driver.o:  \
 		module_bl_mynn.o \
 		module_bl_fogdes.o \
 		module_bl_gwdo.o \
+                module_bl_gwdo_gsl.o \
 		module_bl_temf.o \
```

Comment:
Could we line up this line?

Reply:
I use the "vi" editor and when I view the file it is lined up.  The problem is I created the white spaces with the space bar, whereas it looks like the other lines have indents and tabs to create the white space.  I'm not sure how to make this line consistent with the others from a type-setting standpoint.  Any ideas?

Reply:
Use the vi editor. Put a tab (or whatever character is used above/below).

Reply:
Got it.  Line is lined up now.